web/styles/popovers.css

```diff
@@ -1339,3 +1339,16 @@ ul.navbar-dropdown-menu-outer-list {
         display: block;
     }
 }
+
+.navbar-dropdown-hotkey-hint {
+    color: var(--color-hotkey-hint);
+    text-align: center;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 14px;
+    padding: 2px 5px 3px;
```

Comment:
I noticed that this padding leads to the row being taller than another rows; we need to adjust this CSS so that it doesn't change the spacing between adjacent rows.

Reply:
Oops, fixed.

Reply:
<img width="528" alt="Screenshot 2023-10-21 at 1 37 02 PM" src="https://github.com/zulip/zulip/assets/25124304/50e1d1cb-4d07-46d7-922e-1784fb7de24b">

I didn't realize border width was part of padding too in the design.